downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/Biopax.java

```diff
@@ -0,0 +1,211 @@
+			level3converter.doDump(new String[]{host, database, username, password, port, biopaxDir, speciesName});
+		}
+	}
+	
+	// Next, once all species files for the current BioPAX level have been generated, we must validate them to make sure they are appropriately formatted
+	private static void validateBioPAX(String biopaxDir, int biopaxLevel, String releaseNumber, String outfile) throws Exception {
+		
+		// Rename files, replacing whitespace with underscores
+	    Files.newDirectoryStream(Paths.get(biopaxDir),
+	    	      owlFilepath -> owlFilepath.toString().endsWith(".owl"))
+	    	      .forEach(owlFile -> {
+	    	          String owlFilename = owlFile.toFile().getPath();
+	    	          owlFilename = owlFilename.replaceAll(" +", "_");
+	    	          File formattedOwlFilename = new File(owlFilename);
+	    	          owlFile.toFile().renameTo(formattedOwlFilename);
+	    	      });
+	    // Validate each owl file in the biopax output directory produced by PathwaysConverter
+	    logger.info("Validating owl files...");
+		runBiopaxValidation(biopaxDir);
+
+		// Compress all Biopax and validation files into individual zip files
+		logger.info("Zipping BioPAX" + biopaxLevel + " files...");
+		ZipOutputStream biopaxZipStream = getBiopaxZipStream(biopaxLevel);
+		ZipOutputStream validatorZipStream = getValidatorZipStream(biopaxLevel);
+		Files.newDirectoryStream(Paths.get(biopaxDir),
+			      path -> (path.toString().endsWith(".xml") || path.toString().endsWith(".owl")))
+			      .forEach(f -> {
+			          String spath = f.toFile().getPath();
+			          File file = new File(spath);
+		        	  try {
+		    	          if (spath.endsWith(".owl")) {
+		    	        	writeToZipFile(file, biopaxZipStream);
+		    	          } else if (spath.endsWith(".xml")) {
+							writeToZipFile(file, validatorZipStream);
+		    	          }
+					  } catch (IOException e) {
+						e.printStackTrace();
+					  }
+			      });
+			    biopaxZipStream.close();
+		validatorZipStream.close();
+	    
+	    logger.info("Finished BioPAX");
+	}
+	
+	// Zip utilities
+	private static ZipOutputStream getBiopaxZipStream(int biopaxLevel) throws FileNotFoundException {
+		return getZipOutputStream(biopaxLevel == 2 ? "biopax2.zip" : "biopax.zip");
+	}
+	
+	private static ZipOutputStream getValidatorZipStream(int biopaxLevel) throws FileNotFoundException {
+		return getZipOutputStream(biopaxLevel == 2 ? "biopax2_validator.zip" : "biopax_validator.zip");
+	}
+	
+	private static ZipOutputStream getZipOutputStream(String fileName) throws FileNotFoundException {
+		return new ZipOutputStream(new FileOutputStream(fileName));
+	}
+	
+	//Function for compressing Biopax and validation files
+	  public static void writeToZipFile(File file, ZipOutputStream zipOutputStream) throws IOException {
+			
+		FileInputStream zipInputStream = new FileInputStream(file);
+		ZipEntry zipEntry = new ZipEntry(file.getName().toString());
+		zipOutputStream.putNextEntry(zipEntry);
```

Comment:
Since zipEntry is only used here, it might be good to inline its creation in the putNextEntry method call:
`zipOutputStream.putNextEntry(new ZipEntry(file.getName()));`